README.md

```diff
@@ -30,58 +44,22 @@ The cloudinary_android library is available in [Maven Central](http://repo1.mave
         <version>1.30.0</version>
     </dependency>
 
-
-## Try it right away
-
-Sign up for a [free account](https://cloudinary.com/users/register/free) so you can try out image transformations and seamless image delivery through CDN.
-
-*Note: Replace `demo` in all the following examples with your Cloudinary's `cloud name`.*  
-
-Accessing an uploaded image with the `sample` public ID through a CDN:
-
-    http://res.cloudinary.com/demo/image/upload/sample.jpg
-
-![Sample](https://res.cloudinary.com/demo/image/upload/w_0.4/sample.jpg "Sample")
-
-Generating a 150x100 version of the `sample` image and downloading it through a CDN:
-
-    http://res.cloudinary.com/demo/image/upload/w_150,h_100,c_fill/sample.jpg
-
-![Sample 150x100](https://res.cloudinary.com/demo/image/upload/w_150,h_100,c_fill/sample.jpg "Sample 150x100")
-
-Converting to a 150x100 PNG with rounded corners of 20 pixels:
-
-    http://res.cloudinary.com/demo/image/upload/w_150,h_100,c_fill,r_20/sample.png
-
-![Sample 150x150 Rounded PNG](https://res.cloudinary.com/demo/image/upload/w_150,h_100,c_fill,r_20/sample.png "Sample 150x150 Rounded PNG")
-
-For plenty more transformation options, see our [image transformations documentation](http://cloudinary.com/documentation/image_transformations).
-
-Generating a 120x90 thumbnail based on automatic face detection of the Facebook profile picture of Bill Clinton:
-
-    http://res.cloudinary.com/demo/image/facebook/c_thumb,g_face,h_90,w_120/billclinton.jpg
-
-![Facebook 90x120](https://res.cloudinary.com/demo/image/facebook/c_thumb,g_face,h_90,w_120/billclinton.jpg "Facebook 90x200")
-
-For more details, see our documentation for embedding [Facebook](http://cloudinary.com/documentation/facebook_profile_pictures) and [Twitter](http://cloudinary.com/documentation/twitter_profile_pictures) profile pictures.
-
-
 ## Usage
 
-### Configuration
+### Setup
```

Comment:
Also, would replace the entire text, with steps. "Do 1,2,3" with less explanations. 
Would try to avoid alternate ways of doing the same thing. 
Remember - the main thing here is getting from 0-100 in minimal time. or in other words - having Cloudinary running in your code in no-time. 

Something like:
- Go to console (with link) 
- Copy `CLOUDINARY_URL`
- Paste in `AndroidManifest.xml`
- etc...

Reply:
Of course, at the end we should always put links to 'learn more' in the docs. 
But the important things is that we have a running code in the user's IDE.